workflow/scripts/osemosys_global/transmission/costs.py

```diff
@@ -71,7 +71,8 @@ def get_transmission_costs(df_exist_corrected, df_planned_corrected,
         
         # Variable cost in $/MWh converted to $Million/PJ
         df_var.loc[df_var['tech_group'] == group, 
-                   'VALUE'] = round(var_dict[group] / 3600, 4)
+                   'VALUE'] = round(var_dict[group] / 0.0000036 / 1000000 , 4)
```

Comment:
Can someone confirm that the below change is correct? Assuming variable costs are in $Million/PJ, I think that the previous conversion from $/MWh to $Million/PJ was off by a factor 1000.